pkgs/by-name/ky/kyverno-chainsaw/package.nix

```
@@ -0,0 +1,37 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kyverno-chainsaw";
+  version = "0.1.6";
+  cli = "chainsaw";
+
+  src = fetchFromGitHub {
+    owner = "kyverno";
+    repo = "chainsaw";
+    rev = "v${version}";
+    hash = "sha256-14swkcv70qn1ka0pb3za9a4r15shm1aw93by2k3b1kx3qdar1bcs";
```

When running
```sh
nix run nixpkgs#nixpkgs-review -- pr --post-result 292144
```

 I get the error
 ```
 invalid SRI hash '14swkcv70qn1ka0pb3za9a4r15shm1aw93by2k3b1kx3qdar1bcs'
 ```

And when running
```sh
nix-prefetch-url "https://github.com/kyverno/chainsaw/archive/v0.1.6.tar.gz" \
| xargs nix hash to-sri --type 256
```

 I get a different (see suggestion below) `hash`:
```suggestion
    hash = "sha256-fSt0WxpjtzIuGg+O2x5W9pBHVCuT9uy431jMXkqH6ak=";
```

---

please tell me how to check vendor hash also ?

---

I don't have a better approach at the moment than to use an invalid hash, e.g. `sha256-CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC=` and rebuild 🙁